packages/@okta/i18n/src/properties/login_ok_PL.properties

```diff
@@ -764,12 +764,12 @@ oie.consent.enduser.accept.label = 》Ýéš، îţ´š ɱé ⾼ئ䀕ヸ€홝
 oie.consent.enduser.deny.label = 》Ñö، îţ´š ñöţ ɱé ฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
 oie.consent.enduser.email.allow.description = 》Ţö çöñţîñûé، ŕéţûŕñ ţö ţĥé ðéṽîçé öŕ ŵîñðöŵ ŵĥéŕé ýöû ŕéǫûéšţéð ţĥé éɱåîļ ļîñķ· Ýöû ɱåý çļöšé ţĥîš ŵîñðöŵ åţ åñý ţîɱé· 한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
 oie.consent.enduser.deny.description = 》Ýöû ɱåý çļöšé ţĥîš ŵîñðöŵ åţ åñý ţîɱé· 홝한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
-oie.return.to.original.tab = 》Þļéåšé ŕéţûŕñ ţö öŕîĝîñåļ ţåƀ· 䀕ヸ€홝한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
-oie.device.code.activate.title = 》Åçţîṽåţé ýöûŕ ðéṽîçé €홝한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
-oie.device.code.activate.subtitle = 》Ƒöļļöŵ ţĥé îñšţŕûçţîöñš öñ ýöûŕ ðéṽîçé ţö ĝéţ åñ åçţîṽåţîöñ çöðé ئ䀕ヸ€홝한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
-oie.device.code.activate.label = 》Åçţîṽåţîöñ Çöðé ฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
-oie.device.code.activated.success.title = 》Ðéṽîçé åçţîṽåţéð Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
-oie.device.code.activated.error.title = 》Ðéṽîçé ñöţ åçţîṽåţéð €홝한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
+oie.return.to.original.tab = 》Þļéåšé ŕéţûŕñ ţö ţĥé öŕîĝîñåļ ţåƀ· 한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
```

Comment:
is this change intentional?

Reply:
I ran the tool to generate this, not sure why it changed this line. @haishengwu-okta any thoughts?

Reply:
good catch, fixed it.